e2e-test/src/test/features/BigQueryToNormalize.feature

```diff
@@ -0,0 +1,102 @@
+    When Expand Plugin group in the LHS plugins list: "Transform"
+    When Select plugin: "Transpose" from the plugins list as: "Transform"
+    Then Connect plugins: "BigQuery" and "Transpose" to establish connection
+    When Expand Plugin group in the LHS plugins list: "Sink"
+    When Select plugin: "BigQuery" from the plugins list as: "Sink"
+    Then Connect plugins: "Transpose" and "BigQuery2" to establish connection
+    Then Navigate to the properties page of plugin: "BigQuery"
+    Then Replace input plugin property: "projectId" with value: "projectId"
+    Then Enter input plugin property: "datasetProjectId" with value: "projectId"
+    Then Enter input plugin property: "referenceName" with value: "BQReferenceName"
+    Then Enter input plugin property: "dataset" with value: "dataset"
+    Then Enter input plugin property: "table" with value: "bqSourceTable"
+    Then Click on the Get Schema button
+    Then Validate "BigQuery" plugin properties
+    Then Close the Plugin Properties page
+    Then Navigate to the properties page of plugin: "Transpose"
+    Then Enter Normalize plugin Fields to be Mapped "normalizeBQValidFieldsMapping"
+    Then Enter Normalize plugin Fields to be Normalized "normalizeFieldsToBeNormalizedBQ"
+    Then Select Normalize plugin output schema action: "clear"
+    Then Enter Normalize plugin outputSchema "normalizeBQValidOutputSchema"
+    Then Validate "Transpose" plugin properties
+    Then Close the Plugin Properties page
+    Then Navigate to the properties page of plugin: "BigQuery2"
+    Then Replace input plugin property: "projectId" with value: "projectId"
+    Then Enter input plugin property: "datasetProjectId" with value: "projectId"
+    Then Enter input plugin property: "referenceName" with value: "BQReferenceName"
+    Then Enter input plugin property: "dataset" with value: "dataset"
+    Then Enter input plugin property: "table" with value: "bqTargetTable"
+    Then Click plugin property: "truncateTable"
+    Then Click plugin property: "updateTableSchema"
+    Then Validate "BigQuery2" plugin properties
+    Then Close the Plugin Properties page
+    Then Save the pipeline
+    Then Preview and run the pipeline
+    Then Wait till pipeline preview is in running state
+    Then Open and capture pipeline preview logs
+    Then Verify the preview run status of pipeline in the logs is "succeeded"
+    Then Close the pipeline logs
+    Then Close the preview
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Close the pipeline logs
+    Then Validate OUT record count is equal to records transferred to target BigQuery table
+
+  @BQ_SOURCE_NORMALIZE_TEST @GCS_SINK_TEST
+  Scenario: To verify data is getting transferred from BigQuery to GCS sink plugin successfully with Normalized fields
```

Comment:
no need to test different combinations of sources and sinks. these tests should focus on transpose plugin functionality.

Reply:
Done.